include/WireType.h

```diff
@@ -39,7 +39,9 @@ enum WireType {
     WIRE_TYPE_STRING = 9,
     WIRE_TYPE_TIMESTAMP = 11,
     WIRE_TYPE_DECIMAL = 22,
-    WIRE_TYPE_VARBINARY = 25
+    WIRE_TYPE_VARBINARY = 25,
+    WIRE_TYPE_GEOGRAPHY_POINT = 26, // currenty unsupported
+    WIRE_TYPE_GEOGRAPHY = 27,       // currenty unsupported
```

Comment:
Do these have to match constants in the server or the EE?  If so, should there be a note reminding people of it?